src/app/notifications/qa/events/quality-assurance-events.component.ts

```diff
@@ -0,0 +1,433 @@
+import { hasValue } from '../../../shared/empty.util';
+import { ItemSearchResult } from '../../../shared/object-collection/shared/item-search-result.model';
+import { NotificationsService } from '../../../shared/notifications/notifications.service';
+import {
+  ProjectEntryImportModalComponent,
+  QualityAssuranceEventData
+} from '../project-entry-import-modal/project-entry-import-modal.component';
+import { getFirstCompletedRemoteData } from '../../../core/shared/operators';
+import { PaginationService } from '../../../core/pagination/pagination.service';
+import { Item } from '../../../core/shared/item.model';
+import { FindListOptions } from '../../../core/data/find-list-options.model';
+import {environment} from '../../../../environments/environment';
+
+/**
+ * Component to display the Quality Assurance event list.
+ */
+@Component({
+  selector: 'ds-quality-assurance-events',
+  templateUrl: './quality-assurance-events.component.html',
+  styleUrls: ['./quality-assurance-events.component.scss'],
+})
+export class QualityAssuranceEventsComponent implements OnInit, OnDestroy {
+  /**
+   * The pagination system configuration for HTML listing.
+   * @type {PaginationComponentOptions}
+   */
+  public paginationConfig: PaginationComponentOptions = Object.assign(new PaginationComponentOptions(), {
+    id: 'bep',
+    currentPage: 1,
+    pageSize: 10,
+    pageSizeOptions: [5, 10, 20, 40, 60]
+  });
+  /**
+   * The Quality Assurance event list sort options.
+   * @type {SortOptions}
+   */
+  public paginationSortConfig: SortOptions = new SortOptions('trust', SortDirection.DESC);
+  /**
+   * Array to save the presence of a project inside an Quality Assurance event.
+   * @type {QualityAssuranceEventData[]>}
+   */
+  public eventsUpdated$: BehaviorSubject<QualityAssuranceEventData[]> = new BehaviorSubject([]);
+  /**
+   * The total number of Quality Assurance events.
+   * @type {Observable<number>}
+   */
+  public totalElements$: BehaviorSubject<number> = new BehaviorSubject<number>(null);
+  /**
+   * The topic of the Quality Assurance events; suitable for displaying.
+   * @type {string}
+   */
+  public showTopic: string;
+  /**
+   * The topic of the Quality Assurance events; suitable for HTTP calls.
+   * @type {string}
+   */
+  public topic: string;
+  /**
+   * The rejected/ignore reason.
+   * @type {string}
+   */
+  public selectedReason: string;
+  /**
+   * Contains the information about the loading status of the page.
+   * @type {Observable<boolean>}
+   */
+  public isEventPageLoading: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
+  /**
+   * The modal reference.
+   * @type {any}
+   */
+  public modalRef: any;
+  /**
+   * Used to store the status of the 'Show more' button of the abstracts.
+   * @type {boolean}
+   */
+  public showMore = false;
+  /**
+   * The Open Aire base url for project search
+   */
+  public openAireUrl = environment.qualityAssuranceConfig.openAireUrl;
```

Comment:
not sure why openaire should be available here, shouldn't quality assurance be source agnostic?

Reply:
Hi @paulo-graca , I have adapted the config to use the source id to retrieve the url, so that we would need only a new config for future sources.